src/Administration/Resources/app/administration/src/module/sw-category/page/sw-category-detail/index.js

```diff
@@ -869,7 +869,32 @@ export default {
                 seoUrls.map((seoUrl) => {
                     if (seoUrl.seoPathInfo) {
                         seoUrl.isModified = true;
-                        return this.seoUrlService.updateCanonicalUrl(seoUrl, seoUrl.languageId);
+                        return this.seoUrlService.updateCanonicalUrl(seoUrl, seoUrl.languageId).catch((error) => {
+                            if (error.response && error.response.data && error.response.data.errors) {
+                                error.response.data.errors.forEach((apiError) => {
+                                    const messageKey = `global.error-codes.${apiError.detail}`;
+                                    const params = apiError.meta?.parameters || {};
+                                    const translatedMessage = this.$tc(messageKey, 1, params);
```

Comment:
```suggestion
                                    const translatedMessage = this.$t(messageKey, 1, params);
```

$tc is deprecated. Please adjust it in the other lines too.